recipes/glibmm/all/conanfile.py

```diff
@@ -43,129 +51,118 @@ def _giomm_lib(self):
         return f"giomm-{self._abi_version}"
 
     def validate(self):
-        if hasattr(self, "settings_build") and build.cross_building(self):
+        if hasattr(self, "settings_build") and cross_building(self):
             raise ConanInvalidConfiguration("Cross-building not implemented")
 
         if self.settings.compiler.get_safe("cppstd"):
             if self._abi_version == "2.68":
-                build.check_min_cppstd(self, 17)
+                check_min_cppstd(self, 17)
             else:
-                build.check_min_cppstd(self, 11)
+                check_min_cppstd(self, 11)
+
+        if is_msvc(self) and not self.options.shared:
+            raise ConanInvalidConfiguration("Static library build is not supported by MSVC")
```

Comment:
It's a new limitation? Why did it work previously? Anyway it's a problem to raise for default option value.

Reply:
MSVC static builds are disabled in the official sources, we've renabled them via a patch. I've reinstated support and added a patch for 2.75.0